html-css/week1/mateus_britto/portfolio_page/index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description"
+        content="Unlock the digital potential of your business or non profit with my web development expertise. Allow me to take your online presence to another level.">
+    <link rel="stylesheet" href="styles.css">
+    <title>Mateus Britto - Web Development</title>
+</head>
+
+<body>
+
+    <nav class="nav-bar">
+        <ul class="nav-ul">
+            <li class="nav-li"><a class="section-link" href="#about" aria-label="Learn more about me">About</a></li>
+            <li class="nav-li"><a class="section-link" href="#work" aria-label="Learn more about my work">Work</a></li>
+            <li class="nav-li"><a class="section-link" href="#contact"
+                    aria-label="Learn more about how to contact me">Contact</a></li>
+        </ul>
+    </nav>
+
+    <div class="container">
+        <section id="about">
+
+            <div class="intro">
+                <h2 class="call-to-action">Welcome to my portfolio page!<br><br> Here, I showcase my journey and progress toward becoming a full-stack web developer. <br><br> Grab a cup of coffee and enjoy exploring!</h2>
+            </div>
+            <div class="profile-container">
+                <img class="profile" src="profile.webp" width="480" height="480" alt="profile">
+                <div class="title-container">
+                    <h2 class="introduction-name">Mateus Britto</h2>
+                    <h2 class="introduction-title">Web Developer</h2>
+                </div>
+            </div>
+
+        </section>
+
+        <section id="work">
+            <div class="work-wrapper">
+                <div class="title-wrapper-html">
+                    <h1 class="main-content-html-title">Here are some of my projects</h1>
+                </div>
```

Comment:
YOu can merge those classes and elements into one.

```suggestion
          <h1 class="main-content-html-title">Here are some of my projects</h1>
```